packages/google_maps_flutter/google_maps_flutter_ios/ios/Classes/ClusterManagersController.h

```diff
@@ -0,0 +1,80 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#import <Flutter/Flutter.h>
+#import <Google-Maps-iOS-Utils/GMUClusterAlgorithm.h>
+#import <Google-Maps-iOS-Utils/GMUClusterIconGenerator.h>
+#import <Google-Maps-iOS-Utils/GMUClusterManager.h>
+#import <Google-Maps-iOS-Utils/GMUClusterRenderer.h>
+#import <Google-Maps-iOS-Utils/GMUDefaultClusterIconGenerator.h>
+#import <Google-Maps-iOS-Utils/GMUDefaultClusterRenderer.h>
+#import <Google-Maps-iOS-Utils/GMUGridBasedClusterAlgorithm.h>
+#import <Google-Maps-iOS-Utils/GMUNonHierarchicalDistanceBasedAlgorithm.h>
+#import <Google-Maps-iOS-Utils/GMUSimpleClusterAlgorithm.h>
+#import <Google-Maps-iOS-Utils/GMUStaticCluster.h>
+#import <GoogleMaps/GoogleMaps.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+// Defines cluster managers controller.
+@interface FLTClusterManagersController : NSObject
+
+/**
+ * Initializes FLTClusterManagersController.
+ *
+ * @param methodChannel A Flutter method channel used to send events.
+ * @param mapView A map view that will be used to display clustered markers.
+ */
+- (instancetype)init:(FlutterMethodChannel *)methodChannel mapView:(GMSMapView *)mapView;
```

Comment:
nits: a more Objective-C idiomatic name would be 
`- (instancetype)initWithMethodChannel:(FlutterMethodChannel *)methodChannel mapView:(GMSMapView *)mapView;`

Reply:
Renamed.